utils/deprecations.py

```diff
@@ -0,0 +1,12 @@
+import logging
+from os.path import basename
+
+log = logging.getLogger(__name__)
+
+
+def deprecate_old_command_line_tools(file_name):
+    name = basename(file_name)
+    if name == 'dd-forwarder' or name == 'dogstatsd' or name == 'dd-agent':
```

Comment:
Yep, it worked, but I changed it since it seems that `__file__` is not reliable. Thanks!